account_analytic_no_lines/tests/test_account_analytic_no_lines.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import SavepointCase
+
+
+class TestAccountAnalyticNoLines(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
No stress :smirk: Thank you to you for your investment in the reviewing process of so much PR!!